webapp/src/ts/selectors/index.ts

```diff
@@ -28,7 +28,6 @@ export const Selectors = {
   getShowActionBar: createSelector(getGlobalState, (globalState) => globalState.showActionBar),
   getForms: createSelector(getGlobalState, (globalState) => globalState.forms),
   getFilters: createSelector(getGlobalState, (globalState) => globalState.filters),
-  getIsAdmin: createSelector(getGlobalState, (globalState) => globalState.isAdmin),
```

Comment:
Looks like this isn't used anywhere

Reply:
Yep, I think we moved this functionality to `Session.isOnlineOnly`. This didn't need to be in the state, as it's not subject to change within one load.